src/static/css/bootstrap-table-accordion.css

```diff
@@ -1,123 +1,310 @@
-/************************************
-*
-* Custom Stylings for Bootstrap Table
-* https://github.com/wenzhixin/bootstrap-table
-*
-************************************/
-
-/* restore thicker bootstrap header bottom border removed in bootstrap-table */
+/*=============================================================
+=            Bootstrap Table Accordion Styling Fixes          =
+=         https://github.com/wenzhixin/bootstrap-table        =
+
+=  In Bootstrap 3, many important styling rules for the
+=  tables are missing, such as danger, warning, info etc.
```

Comment:
I don't think this is the case.  For example, the danger styling works fine in the existing profile list view that uses bootstrap-table.  Better to use existing styles if possible to remove unnecessary code and styling (everything we customize makes maintaining the project a little bit harder)

![image](https://github.com/bytedeck/bytedeck/assets/10604391/383f42bd-b32c-4629-9b9d-6fab4822cb36)


The color is coming from default bootstrap css:

![image](https://github.com/bytedeck/bytedeck/assets/10604391/cb3bb9d8-6ddc-43f6-89d6-5c079a24c4e2)

Reply:
You were right. Bootstrap had `danger`, `warning`, `info`, and `success` classes for table rows. It looks like it's missing muted classes, but `panel-muted` ended up working perfectly. Not sure why I can't get `panel-warning`, `panel-danger`, etc to work.

I also couldn't figure out a way to have the danger/warning/info border around an expanded row, so I kept the styles for that:
```css
  /* Updating border-color based on class */
  /* Specify the colors for each class */
  .accordian-table > tbody > tr {
    --accordian-table-border-color: transparent;
  }

  .accordian-table > tbody > tr > .default {
    /* make what it would be normally */
    --accordian-table-border-color: #337ab7;
  }

  .accordian-table > tbody > tr > .muted {
    --accordian-table-border-color: #333;
  }

  .accordian-table > tbody > tr > .warning {
    --accordian-table-border-color: #8a6d3b;
  }

  .accordian-table > tbody > tr > .danger {
    --accordian-table-border-color: #a94442;
  }

  .accordian-table > tbody > tr > .info {
    --accordian-table-border-color: #31708f;
  }


  /* Updating list-group-item border-color to match the row */
  tr.detail-view > td > div > ul > .list-group-item,
  tr.detail-view > td > div > .list-group-item {
    border-left-color: var(--accordian-table-border-color);
    border-right-color: var(--accordian-table-border-color);
  }

  tr.detail-view > td > div > ul > .list-group-item:last-child,
  tr.detail-view > td > div > .list-group-item:last-child  {
    border-bottom-color: var(--accordian-table-border-color);
  }
  ```